### ISSUE TITLE
Installation Failed because Unable to Find ft2build.h and zlib.h

### ISSUE THREAD

**Issue by imk1** [ImageMagick/ImageMagick#1549]
### Prerequisites

- [x] I have written a descriptive issue title
- [x] I have verified that I am using the latest version of ImageMagick
- [x] I have searched [open](https://github.com/ImageMagick/ImageMagick/issues) and [closed](https://github.com/ImageMagick/ImageMagick/issues?q=is%3Aissue+is%3Aclosed) issues to ensure it has not already been reported

### Description

I tried to install ImageMagick.  The installation failed with the following error:
MagickCore/annotate.c:89:10: fatal error: ft2build.h: No such file or directory
 #include <ft2build.h>
          ^~~~~~~~~~~~
compilation terminated.
Eventually, I figured out that ft2build.h was on my cluster.  I re-ran configure with CPPFLAGS pointing to the directory with ft2build.h.  Installation then failed with this error:
MagickCore/blob.c:76:10: fatal error: zlib.h: No such file or directory
 #include "zlib.h"
          ^~~~~~~~
compilation terminated.
I discovered that zlib.h was also on my cluster.  I then re-ran configure with CPPFLAGS pointing to both the directory with ft2build.h and the directory with zlib.h.  At this point, make finally ran to completion.  Although I do not need help, it occurred to me that others might have this problem and benefit from the solution that I found.

### Steps to Reproduce

First time:
./configure
make
Second time:
./configure CPPFLAGS='-I[DIRECTORY WITH ft2build.h]'
make
Third (successful) time:
./configure CPPFLAGS='-I/[DIRECTORY WITH ft2build.h] -I[DIRECTORY WITH zlib.h]'

### System Configuration

Linux cluster with Centos 6 (I am not root)

- ImageMagick version: 7.0.8
- Environment (Operating system, version and so on): Linux cluster with Centos 6
- Additional information: I do not have sudo permissions

**Comment by dlemstra** [comment]
@imk1 I am guessing that you figured out why this was failing. Could you add an explanation for someone who finds this issue in the future?

**Comment by imk1** [comment]
The issue is that, for some reason, the compiler could not find where some of the dependencies were on the cluster that I am using.  I fixed it by adding their paths when I ran configure.

I also had to add LDFLAGS to point to some libraries that the compiler was not able to find automatically.  Therefore, my entire configure command looked like this:
./configure CPPFLAGS='-I/[DIRECTORY WITH ft2build.h] -I[DIRECTORY WITH zlib.h] -I[DIRECTORY WITH fontconfig/fontconfig.h]' LDFLAGS='-L[directory with freetype2 library] -L[directory with libpng library] -L[directory with fontconfig library] -L[directory with zlib library]'
